manage/installing/installation.rst

```diff
@@ -125,11 +125,11 @@ Download the latest Plone unified installer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Download from `the plone.org download page <https://plone.org/download>`_ to your server using wget command. Curl also works.
```

Comment:
Should we open a own ticket for this issue ?

@tkimnguyen it would make sense to also add links to the versions, like 4.3.8 -> https://launchpad.net/plone/...

otherwise ppl would not be able to find the download of the version they want to use, right ?